R/R/install.R

```diff
@@ -0,0 +1,84 @@
+#' Install Metaflow Python package
+#'
+#' This function wraps installation functions from [reticulate][reticulate::reticulate] to install the Python packages
+#' **metaflow** and it's Python dependencies.
+#'
+#' This package uses the [reticulate][reticulate::reticulate] package
+#' to make an interface with the [Metaflow](https://metaflow.org/)
+#' Python package. 
+#'
+#' @param method `character`, indicates to use `"conda"` or `"virtualenv"`
+#' @param envname `character`, name of environment into which to install
+#' @param version `character`, version of Metaflow to install. For general use of this package,
+#'   this is set automatically, so you should not need to specify this.
+#' @param ... other arguments sent to [reticulate::conda_install()] or
+#'    [reticulate::virtualenv_install()]
+#'
+#' @seealso
+#' [reticulate: Using reticulate in an R Package](https://rstudio.github.io/reticulate/articles/package.html),
+#' [reticulate: Installing Python Packages](https://rstudio.github.io/reticulate/articles/python_packages.html)
+#' @examples
+#' \dontrun{
+#' # not run because it requires Python
+#' install_metaflow()
+#' }
+#' @export
+install_metaflow <- function(method = c("conda", "virtualenv"),
```

Comment:
Can we rename this to `install` to ensure backwards compatibility?

Reply:
I think it may be worth it to switch since it's so early in the package's development but can you definitely overrule me on this!

`install` has a namespace issue with the popular devtools package and there is a strong consensus on the `install_` naming convention for wrapping python projects (I can't think of one that doesn't).

Reply:
Okay, let's go with `install_`